### DISCUSSION TITLE
System.Exception: '6'

### DISCUSSION THREAD

**Discussion by pd2871** [open-mmlab/mmdeploy#1732] [Q&A] [answered]
@irexyc, I need your help for one more time,  the error is coming unexpectedly, `System.Exception: '6'`, Can you help me on this one as well? Thanks.
 
![Screenshot (2809)](https://user-images.githubusercontent.com/59336638/217498040-64a6ea8f-8f24-4c0b-a562-bb615d38345e.png)
=============================================================================

![Screenshot (2807)](https://user-images.githubusercontent.com/59336638/217498405-216c7e3d-c079-46cf-93a4-16c060cb238f.png)

**Comment by irexyc** [comment]
Hi @pd2871,  when you run the program, there will be a black windows showing the processing log, could you provide the full log?

**Comment by pd2871** [comment]
@irexyc  you mean this one?

![Screenshot (2813)](https://user-images.githubusercontent.com/59336638/217501583-208d1197-40f9-448d-bdbf-faed29e8df29.png)

**Comment by irexyc** [comment]
Hi @pd2871, have no clue for this error. Is it convenient to share the model and test image?

**Reply by pd2871** [reply]
@irexyc  sure, Here is the link of model and image https://drive.google.com/file/d/1Fy8Vd9mUSpQgJQVohsOJjnuNiXap3xCf/view?usp=sharing

**Comment by irexyc** [comment, marked as answer]
Hi @pd2871 , I think the problem is you used the wrong handle. The detail.json tells me that you model is a detection model, therefore you should use `Detector` instead of `Segmentor`

**Comment by pd2871** [comment]
Thanks @irexyc :)))
